.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+AllCops:
```

Comment:
I use these settings in all of my projects.
About the `exclude` settings down below, they really doesn't make sense.